aio/aio-proxy/aio_proxy/parsers/nature_juridique.py

```diff
@@ -0,0 +1,23 @@
+from typing import List
+
+
+def validate_nature_juridique(list_nature_juridique_clean: List[str]):
+    """Check the validity of list_section_activite_principale.
+
+    Args:
+        list_nature_juridique_clean(list(str), optional):
+        list_nature_juridique extracted and cleaned.
+
+    Returns:
+        None if list_nature_juridique_clean is None.
+        list_nature_juridique_clean if valid.
+
+    Raises:
+        ValueError: if one of the values in list_nature_juridique_clean is not valid.
+    """
+    if list_nature_juridique_clean is None:
+        return None
+    for nature_juridique in list_nature_juridique_clean:
+        if len(nature_juridique) != 4 or not nature_juridique.isdigit():
+            raise ValueError("Au moins une nature juridique est non valide.")
```

Comment:
Pourquoi pas retourner la liste des codes valides comme pour les départements ?

Reply:
From what I understand in this page: there is a 9*37*260 possible values which seems like too many values to return don't you think ?

Reply:
I can add a more detailed check with list of values.

Reply:
Done !